query-connector/src/app/(pages)/fhirServers/page.tsx

```diff
@@ -48,19 +49,23 @@ const FhirServers: React.FC = () => {
   const [modalMode, setModalMode] = useState<ModalMode>("create");
   const modalRef = useRef<ModalRef>(null);
 
-  // Handle client-side hydration
-  useEffect(() => {
-    setIsClient(true);
-  }, []);
-
   // Fetch FHIR servers
-  useEffect(() => {
-    if (isClient) {
-      getFhirServerConfigs(true).then((servers) => {
+  async function fetchFHIRServers() {
+    try {
+      await getFhirServersList(true).then((servers) => {
         setFhirServers(servers);
```

Comment:
i think we don't need the `.then` here anymore since we're using `await`

Reply:
done!